tests/pytests/unit/runners/test_doc.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+import salt.runner
+import salt.runners.doc as doc
+from tests.support.mock import PropertyMock, patch
+
+
+@pytest.fixture(autouse=True)
+def setup_loader(request):
```

Comment:
This needs to be updated to the new way of setting loader modules.
Please check the other tests on how.